spaces/aliases.json

```
@@ -1,5 +1,6 @@
-{
-  "aave": "aave.eth",
+{ "name" :"real_name":"Aisha Redl-Sherwood"
+ "alias":"cloudfare":"cryptob3auty.com"
+  "aave":"aave.eth",
```

Your domain field value should be similar to your space directory `"my.custom.domain": "my-space.eth"`.
```suggestion
{ 
  "aave":"aave.eth",
```